src/modules/src/commander.c

```diff
@@ -37,7 +37,7 @@
 #include "debug.h"
 #include "ext_position.h"
 
-#define MIN_THRUST  1000
+#define MIN_THRUST  0

```

Comment:
Yep good question. The brushed motors don't start spinning until ~3000 so I think it was initially intended so that power isn't consumed without any movement. When it spinns it can go lower so that is were 1000 comes from.